js/modules/inputValidation.js

```diff
@@ -0,0 +1,108 @@
+const forms = Array.from(document.forms);
+let isInputHandlerAdded = false;
+
+const REGEXP = {
+	NAME: /^[а-яА-ЯёЁa-zA-ZЁёЇїІіЄєҐґ']+$/,
+	PHONE: /^(?! )^[+\-\s()0-9]{10,20}$/,
```

Comment:
Why do you need this?
![image](https://user-images.githubusercontent.com/37267977/223636533-35d249b3-9a21-4c69-950f-0c5e8d8b7823.png)

Reply:
It doesn't allow space at the beginning of the string